library/src/main/java/com/google/android/exoplayer2/text/eia608/Eia608Decoder.java

```diff
@@ -159,23 +178,79 @@
     0xC5, 0xE5, 0xD8, 0xF8, 0x250C, 0x2510, 0x2514, 0x2518
   };
 
+  // Maps EIA-608 PAC row numbers to WebVTT cue line settings.
+  // Adapted from: https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#x1-preamble-address-code-pac
+  private static final float[] CUE_LINE_MAP = new float[] {
```

Comment:
If you want to use a map and the standard has the row values in specific, but not linear order like: (1, 3, 2).

And than you reorder everything with a switch case, like you map the value 0x1 to the index 4:

case 0x1:  // Row 5 or 6
   cueLine = CUE_LINE_MAP[4];

Then you could put those values into the map in the same order as the standard specifies, and than you can remove the switch and directly index the map:
cueLine[ 0x1 ];

See previous implementation:
https://github.com/google/ExoPlayer/pull/1312/files#diff-babdf8243982d21611d6cb14f7df1bf9R104